tests/components/ozw/test_light.py

```diff
@@ -149,3 +149,151 @@ async def test_light(hass, light_data, light_msg, sent_messages):
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#ffbb370000", "ValueIDKey": 659341335}
+
+    # Test setting white_value
+    new_color = 215
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "white_value": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 17
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#000000d700", "ValueIDKey": 659341335}
+
+    # Test setting rgb_color with white_value
+    new_color = 215
+    rgb_color = [192, 92, 35]
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {
+            "entity_id": "light.led_bulb_6_multi_colour_level",
+            "white_value": new_color,
+            "rgb_color": rgb_color,
+        },
+        blocking=True,
+    )
+    assert len(sent_messages) == 19
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#ff7a2e0000", "ValueIDKey": 659341335}
+
+
+async def test_no_rgb_light(hass, light_no_rgb_data, light_msg, sent_messages):
+    """Test setting up config entry."""
+    await setup_ozw(hass, fixture=light_no_rgb_data)
+
+    # Test loaded no white level support
+    state = hass.states.get("light.master_bedroom_l_level")
+    assert state is not None
+    assert state.state == "off"
+
+
+async def test_no_ww_light(hass, light_no_ww_data, light_msg, sent_messages):
+    """Test setting up config entry."""
+    await setup_ozw(hass, fixture=light_no_ww_data)
+
+    # Test loaded no ww support
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "off"
+
+
+async def test_no_cw_light(hass, light_no_cw_data, light_msg, sent_messages):
+    """Test setting up config entry."""
+    await setup_ozw(hass, fixture=light_no_cw_data)
+
+    # Test loaded no cw support
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "off"
+
+
+async def test_wc_light(hass, light_wc_data, light_msg, sent_messages):
+    """Test setting up config entry."""
+    await setup_ozw(hass, fixture=light_wc_data)
+
+    # Test loaded no white LED support
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "off"
```

Comment:
We should test a message where the light is on too and check that the color state attributes are set correctly.

Reply:
Got it in the latest commit

Reply:
I added tests to the WW / CW only tests too